### ISSUE TITLE
Topic posts text

### ISSUE THREAD

**Issue by nightw0lv** [pfitzer/kunena-discord#14]
when a topic is created
A new thread was opend in our forum. link etc..
now when a post is done on that topic
A new thread was opend in our forum.
same message

**Comment by pfitzer** [comment]
Yes at the moment always the text from the first message is shown. I will try to fix this.

**Comment by ailocksmith** [comment]
Just chipping in that I also noticed this bug. Would be great if you find a solution. I'll buy you a beer now because I'm already happy with this plugin

**Comment by pfitzer** [comment]
Ok, this comes from missing post id as html anker in url. Think i got the fix, but have to test this with one of my live sites.

@Jeoff-ke thx for the beer ;)

**Comment by JaesicStormsong** [comment]
Quick note that while not a 'bug' per se, the message sent to Discord is:

A new thread was opend in our forum. XXXXX XXX X XXXXX Link

where XXX's are the name of the thread in bold and Link is an active link back to the thread in a browser. Also, the initial wording is in italics.

So... you have a 'typo' going out to Discord ever time it posts (opend should be opened). And if you're correcting the typo, I'd suggest modifying the format to:

A new thread was opened in our forum: XXXX XXX XXX XXXX -> Link

Otherwise, good plugin. If it works as effectively as it seems from first tests; I'll be using it to push problem reports and critical / time sensitive forums posts to Discord on my phone to serve as an alert system. 

Do you have any further plans to add more features to it?

**Comment by nightw0lv** [comment]
you can say that this plugin is abandoned from the time beign until today

**Comment by JaesicStormsong** [comment]
> you can say that this plugin is abandoned from the time beign until today

That's too bad. It's a good plugin as is.

It would be even nicer if it weren't locked into ten hooks, and if it were possible to clarify in the discord posts whether it were a new thread or a post within an existing thread. 

Another feature that would be a nice addition would be a check box and 'name' box for each hook that would let you add an alert to specific users in the discord channel. For example, clicking to activate and putting in Jaesic and the discord push adds a @Jaesic to the message, so Discords notification signal yells at me. :)

Those things would make it super-valuable to admins/moderators.
